torch/nn/quantized/modules/linear.py

```diff
@@ -0,0 +1,65 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+import torch
+from ...modules.linear import Linear as NNLinear
+from ...._jit_internal import weak_module
+
+@weak_module
+class Linear(NNLinear):
+    r"""
+    A quantized linear module with quantized tensor as inputs
+    and outputs.
+    We adopt the same interface as `torch.nn.Linear`, please see https://pytorch.org/docs/stable/nn.html#torch.nn.Linear
+    for documentation.
+
+    Similar to `torch.nn.Linear`, attributes will be randomly initialized at
+        module creation time and will be overwritten later
+
+    Attributes:
+        weight: the non-learnable quantized weights of the
+                module which are of shape :math:`(\text{out\_features}, \text{in\_features})`.
+        bias:   the non-learnable bias of the module of shape :math:`(\text{out\_features})`.
+                If :attr:`bias` is ``True``, the values are initialized to zero.
+        out_scale: `scale` parameter of output Quantized Tensor, type: float
+        out_zero_point: `zero_point` parameter for output Quantized Tensor, type: long
+
+    Examples::
+
+        >>> m = nn.quantized.Linear(20, 30)
+        >>> input = torch.randn(128, 20)
+        >>> output = m(input)
+        >>> print(output.size())
+        torch.Size([128, 30])
+    """
+    __constants__ = ['bias', 'in_features', 'out_features']
+
+    def __init__(self, in_features, out_features, bias=True):
+        assert bias, 'nobias is not supported Quantized Linear module yet'
```

Comment:
We should handle case with bias=False. In this case, there will be no self.bias to delete. We would still create qbias and set it to all zeros.

Reply:
you mean just accept the argument and keep the behavior the same?

Reply:
if we use this hack then quantized module will always have bias after deserialization even when user specify bias=False.

I think we can only support it if fbgemm has nobias implementation

Reply:
Agreed.